src/core/MOM.F90

```diff
@@ -2435,6 +2462,12 @@ subroutine initialize_MOM(Time, Time_init, param_file, dirs, CS, restart_CSp, &
       CS%tv%S => CS%S
     endif
 
+    ! Reset the first direction if it was found in a restart file.
+    if (CS%first_dir_restart > -0.5) &
```

Comment:
Is the value of `-0.5` significant here?  Or is it just something greater than `-1.0`?  A comment might be a good idea here (although not necessary).

Reply:
The value was set to -1.0 before it was read from the restart file, so the comment above does accurately explain what the code is doing here, even though it does not explain why I used -0.5 instead of -1.0.  The value of -0.5 here is to avoid a comparison with exactly the same value as it was set to, but it probably could be `> -1.0` or it could be `>= 0.0` instead.  If it were an integer instead of a real value, it would have to become one of these other expressions.

Reply:
One of the inequalities might be more clear, but it's not necessary; I'll merge this.